### ISSUE TITLE
Python error after reading MIDI file

### ISSUE THREAD

**Issue by solstice-r** [Cantersoft/YTPMVE#10]
On Vegas 18, I get the following after launching the .cs file:
```
Traceback (most recent call last):
  File "YTPMVE v0.110 Py.py", line 31, in <module>
 NameError: name 'exit' is not defined
 [17280] Failed to execute script YTPMVE v0.110 Py
 ```
Afterwards all events in the timeline are deleted, also why would it launch an exe if I have python and mido?

**Comment by Cantersoft** [comment]
It should only launch the executable if the Python script is missing. Your fork solved this issue, correct?

**Comment by solstice-r** [comment]
Fixed with #11
